### PR TITLE
Access iframe: resolve config vars, define authorization better, docs

### PR DESCRIPTION
**Pull request by dvoytenko**
Partial for #13287.

/cc @rudygalfi

### CODE REVIEW

File: extensions/amp-access/0.1/iframe-api/README.md [hunk 1]
```diff
@@ -42,3 +60,47 @@ class Controller {
...
+    "SOURCE_URL": "https://example.org/doc1.amp",
+    "DOCUMENT_REFERRER": "https://other.com"
+  }
+}
+```
+
+
+## Authorize method
+
+The `authize` method checks whether the user should be able to access this document. It's expected to be in the following format:
```
Comment (rudygalfi) [inline extensions/amp-access/0.1/iframe-api/README.md]: Should be "`authorize` method"?

File: extensions/amp-access/0.1/amp-access-iframe.js [hunk 2]
```diff
@@ -108,11 +114,51 @@ export class AccessIframeAdapter {
...
+    return this.connect().then(() => {
       return this.messenger_.sendCommandRsvp('pingback', {});
     });
   }
 
+  /**
+   * @return {!Promise}
+   * @package Visible for testing only.
+   */
+  connect() {
```
Comment (prateekbh) [inline extensions/amp-access/0.1/amp-access-iframe.js]: tests for `connect` and `resolveConfig_` ?

File: extensions/amp-access/0.1/iframe-api/README.md (hunk 1 above)
Comment (dvoytenko) [reply extensions/amp-access/0.1/iframe-api/README.md]: Done :)

File: extensions/amp-access/0.1/amp-access-iframe.js (hunk 2 above)
Comment (dvoytenko) [reply extensions/amp-access/0.1/amp-access-iframe.js]: The test section for both is in the `test-amp-access-iframe.js` within the `describe('runtime connect', () => {...}`. PTAL.

**Pull request merged**